### PR TITLE
Updates to Step-One and Step-Two

### PR DESCRIPTION
**Pull request by ColonelBlimp**
Mostly, wording, layout, etc.  Some small code changes from implementations to interfaces.

### CODE REVIEW

File: Step-One.md [hunk 1]
```diff
@@ -11,8 +11,15 @@ of the running example is available in the
...
-of the `org.eclipse.handly.examples.basic.ui` bundle:
+in the package:
+
+`org.eclipse.handly.examples.basic.ui.model`
+
+of the
+
+`org.eclipse.handly.examples.basic.ui`
+
+bundle:
```
Comment (pisv) [inline Step-One.md]:
```suggestion
in the package

    org.eclipse.handly.examples.basic.ui.model

of the

    org.eclipse.handly.examples.basic.ui

bundle:
```

File: Step-One.md [hunk 2]
```diff
@@ -26,8 +33,11 @@ public interface IFooProject
 
 There are no methods yet, just the blank interface declarations.
 
-Let's define the corresponding implementation classes in the package
-`org.eclipse.handly.internal.examples.basic.ui.model` of the same bundle:
+Let's define the corresponding implementation classes in the package:
+
+`org.eclipse.handly.internal.examples.basic.ui.model`
+
+of the same bundle:
```
Comment (pisv) [inline Step-One.md]:
```suggestion
Let's define the corresponding implementation classes in the package

    org.eclipse.handly.internal.examples.basic.ui.model

of the same bundle:
```

File: Step-Two.md [hunk 3]
```diff
@@ -9,10 +9,15 @@ down to structural elements inside source files. The complete source code
...
-these interfaces extend the relevant 'extension interfaces' such as
+As usual, we begin by defining the interfaces for the new model elements in the package:
+
+`org.eclipse.handly.examples.basic.ui.model`
+
+of the
+
+`org.eclipse.handly.examples.basic.ui`
+
+bundle. We will make these interfaces extend the relevant 'extension interfaces' such as
```
Comment (pisv) [inline Step-Two.md]:
```suggestion
As usual, we begin by defining the interfaces for the new model elements in the package

    org.eclipse.handly.examples.basic.ui.model

of the

    org.eclipse.handly.examples.basic.ui

bundle. We will make these interfaces extend the relevant 'extension interfaces' such as
```

File: Step-Two.md [hunk 4]
```diff
@@ -75,8 +80,11 @@ public interface IFooDef
 }
 ```
 
-The corresponding implementation classes are defined in the package
-`org.eclipse.handly.internal.examples.basic.ui.model` of the same bundle:
+The corresponding implementation classes are defined in the package:
+
+`org.eclipse.handly.internal.examples.basic.ui.model`
+
+of the same bundle:
```
Comment (pisv) [inline Step-Two.md]:
```suggestion
The corresponding implementation classes are defined in the package

    org.eclipse.handly.internal.examples.basic.ui.model

of the same bundle:
```

File: Step-One.md [hunk 5]
```diff
@@ -400,7 +411,9 @@ class FooModelCache
         else if (element instanceof IFooProject)
             return projectCache.get(element);
         else
+        /* BUG
```
Comment (pisv) [inline Step-One.md]:
```suggestion
```

File: Step-One.md [hunk 6]
```diff
@@ -400,7 +411,9 @@ class FooModelCache
...
             return null;
+         */
```
Comment (pisv) [inline Step-One.md]:
```suggestion
```

**Review by pisv — COMMENTED** [review]
Thanks a lot for the thorough revision! My suggestions mostly have to do with proposed changes to layout. After some experiments using the actual wiki, I have found an approach that seems to work slightly better, although I must say I'm not a big fan of it either. Probably, the text needs to be re-written to avoid this layout issue altogether.

**Comment by ColonelBlimp** [comment]
Go for it. I don't spend a lot of time writing for GitHub wiki. I just
clarified what required me to take a second or third look...

**Comment by ColonelBlimp** [comment]
I will continue with the other pages soon 😊

**Comment by pisv** [comment]
> I don't spend a lot of time writing for GitHub wiki.

Me too 😊

> I will continue with the other pages soon 😊

Thank you for contributing!

**Pull request merged**